_imports/colors.less

```diff
@@ -56,6 +56,7 @@
 @color-button-green: rgb(76, 201, 71); /* this is a green */
 @color-button-maroon: rgb(202, 60, 60); /* this is a maroon */
 @color-button-light-blue: rgb(95, 198, 218); /* this is a light blue */
+@list-item-background-hover: #ffc;
```

Comment:
this file should only be colors. I'd define this variable inside of `emulate_user.less` as it is more local to that only

Reply:
👍